neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -97,6 +98,7 @@ public virtual Fixed8 NetworkFee
         protected Transaction(TransactionType type)
         {
             this.Type = type;
+            this.RecivedTimestamp = DateTime.UtcNow;
```

Comment:
Have you considered to store a timestamp as long field? [DateTimeOffset.ToUnixTimeSeconds](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.tounixtimeseconds)